### DISCUSSION TITLE
How to integrate gtest to cmake project?

### DISCUSSION THREAD

**Discussion by ugurkebir** [MaJerle/stm32-cube-cmake-vscode#14] [Q&A] [answered]
Hello,

I converted the CubeIDE project to cmake project by following the steps in the repository, but I want to add it to my project in gtest. How can I do that?

**Comment by MaJerle** [comment, marked as answer]
This repository let's you go to CMake from CubeIDE/MX. What you do in CMake on top, is not part of the lib.

Sorry, but I am not sure I can help here.

**Reply by ugurkebir** [reply]
Ok Mr. Majerle, thanks for answer. Do you have any advice?
